src/Components/test/testassets/BasicTestApp/ShowUriComponent.razor

```diff
@@ -0,0 +1,3 @@
+@page "/show-uri"
+@inject IUriHelper UriHelper
+The current URL is <strong>@UriHelper.GetAbsoluteUri()</strong>
```

Comment:
Is it worth considering/explaining that we won't trigger a navigation (OnLocationChanged) when you connect? For those doing something hash-based, I can see that being the next thing that gets asked for.

Reply:
I understand what that would mean with stateful prerendering, but I don't know what it would mean for the cases we're supporting in 3.0. With stateless prerendering, your entire app is restarting, so would people really expect some kind of navigation event to occur? Everything is being reset.

Reply:
yeah good point. At the most it would be a docs update, and now it's not needed at all.